spirv-tools/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+shopt -s nullglob
+
+declare -A targets=(
+  [x86_64-linux]="musl64"
+  [i686-linux]="musl32"
+  [aarch64-linux]="aarch64-multiplatform-musl"
+  [windows-x86]="mingw32"
+  [windows-x64]="mingwW64"
+)
+
+for target_dir in "${!targets[@]}"; do
+  read -r out
+  mkdir -p "./$target_dir/bin"
+  cp "$out"/bin/{spirv-dis*,spirv-val*,*.dll} "./$target_dir/bin"
+  chmod -R u+w "$target_dir"
+done < <(nix build --print-out-paths --no-link "${targets[@]/#/.#cross.}")
```

Comment:
This script didn't work as intended on my WSL.
The last line 19 caused a syntax error.
I like to suggest a following script for an easier read and maintainance.
```
#!/usr/bin/env bash

targets="x86_64-linux:musl64"
targets+=" i686-linux:musl32"
targets+=" aarch64-linux:aarch64-multiplatform-musl"
targets+=" windows-x86:mingw32"
targets+=" windows-x64:mingwW64"

for dst_target in $targets
do
        dst="$(echo "$dst_target" | cut -d: -f1)"
        target="$(echo "$dst_target" | cut -d: -f2)"

        echo "Building for $dst_target ..."
        out="$(nix build --print-out-paths --no-link ".#cross.$target")"

        echo "Copying binaries from $out to $dst ..."
        mkdir -p "$dst/bin"
        for f in "$out/bin/"{spirv-dis*,spirv-val*,*.dll}
        do
                if [ -f "$f" ]
                then
                        cp "$f" "$dst/bin"
                fi
        done
        chmod -R u+w "$dst"
done
```